src/components/PostEditor/PostEditor.js

```diff
@@ -565,13 +565,13 @@ export default class PostEditor extends React.Component {
         <div styleName='header'>
           <div styleName='initial'>
             <div>
-              <Button {...this.postTypeButtonProps(type)} />
+              <Button noDefaultStyles {...this.postTypeButtonProps(type)} />
```

Comment:
im confused about what changed here that requires this

Reply:
I am also confused why this requires the default styles to be off.